app/controllers/concerns/foreman/controller/puppet/hosts_controller_extensions.rb

```diff
@@ -1,13 +1,9 @@
 module Foreman::Controller::Puppet::HostsControllerExtensions
   extend ActiveSupport::Concern
 
-  PUPPETMASTER_ACTIONS = [:externalNodes, :lookup]
```

Comment:
Just asking here since it was moved to the foreman_puppet plugin, but what is `:lookup`? I know `:externalNodes` but I can't really find out what `:lookup` is. Is it something that foreman_puppet can safely drop?

Reply:
I believe so, I didn't find it in routes nor in any controller actions :shrug:

Reply:
I believe I've traced it back to 0265427b9cb814dabf371099351a9ba5af837c3b which introduced it. However, I don't really understand why it adds `:lookup` as a `before_filter` there.

Reply:
Interesting. it might have been wrong since the day 1 and noone have given it a thought ever since. Not even me while moving it around :):shrug:

Reply:
Just so we can mark this comment as resolved: https://github.com/theforeman/foreman_puppet/pull/245